### PR TITLE
SAK-42122 Implementation of standardized banners in Portal, RSF, and Lessonbuilder

### PR DESCRIPTION
**Pull request by kjw47**
https://jira.sakaiproject.org/browse/SAK-42122

Replaced various banners in rsf, lessonbuilder, and portal with new banners created for SWITCH in SAK-41866. Also removed some unnecessary inline styles attached to the banners.

### CODE REVIEW

File: lessonbuilder/tool/src/webapp/templates/ShowPage.html [hunk 1]
```diff
@@ -1310,9 +1307,8 @@ <h3 class="questionText" rsf:id="questionText"></h3>
 			</div>
 			<div id="add-multimedia-dialog" rsf:id="add-multimedia-dialog" class="lessonsDialog">
 				<div class="ui-widget" id="mm-error-container">
-					<div class="messageError">
-						<p><span class="ui-icon ui-icon-alert" style="float: left; margin-right: .3em;"></span> 
-						<strong rsf:id="msg=simplepage.error_colon"></strong> <span id="mm-error"></span></p>
+					<div class="sak-banner-error">
+						<p><strong rsf:id="msg=simplepage.error_colon"></strong> <span id="mm-error"></span></p>
```
Comment (adrianfish) [inline lessonbuilder/tool/src/webapp/templates/ShowPage.html]: space

File: lessonbuilder/tool/src/webapp/templates/ShowPage.html [hunk 2]
```diff
@@ -1735,9 +1731,8 @@ <h3 class="questionText" rsf:id="questionText"></h3>
 			
             <div id="movie-dialog" rsf:id="movie-dialog" class="lessonsDialog">
 				<div class="ui-widget" id="movie-error-container">
-					<div class="messageError">
-						<p><span class="ui-icon ui-icon-alert" style="float: left; margin-right: .3em;"></span> 
-						<strong rsf:id="msg=simplepage.error_colon"></strong> <span id="movie-error"></span></p>
+					<div class="sak-banner-error">
+						<p><strong rsf:id="msg=simplepage.error_colon"></strong> <span id="movie-error"></span></p>
```
Comment (adrianfish) [inline lessonbuilder/tool/src/webapp/templates/ShowPage.html]: :) .... space

File: lessonbuilder/tool/src/webapp/templates/ShowPage.html [hunk 3]
```diff
@@ -2077,9 +2071,8 @@ <h3 class="questionText" rsf:id="questionText"></h3>
 			</div>
 			<div id="question-dialog" rsf:id="question-dialog" class="lessonsDialog">
 			  <div class="ui-widget" id="question-error-container">
-			    <div class="messageError">
-			      <p><span class="ui-icon ui-icon-alert" style="float: left; margin-right: .3em;"></span> 
-				<strong rsf:id="msg=simplepage.error_colon"></strong> <span id="question-error"></span></p>
+			    <div class="sak-banner-error">
+			      <p><strong rsf:id="msg=simplepage.error_colon"></strong> <span id="question-error"></span></p>
```
Comment (adrianfish) [inline lessonbuilder/tool/src/webapp/templates/ShowPage.html]: unneeded space

**Comment by fostersdesign** [comment]
@adrianfish, are you okay with @kjw47's latest changes?

**Pull request merged**